releases/index.md

```diff
@@ -142,3 +278,9 @@ This calendar versioning scheme began with v19.1. Prior releases use a different
 - A major release is produced twice a year indicating major enhancements to product functionality. A change in the `YY.R` component denotes a major release.
 
 - A patch (or maintenance) release is produced to roll out critical bug and security fixes. A change in the `PP` component denotes a patch release.
+
+<script>
+function ExperimentalAlert() { 
+    alert('This experimental download is not yet qualified for production use.');
```

Comment:
nit: s/`experimental download`/`experimental database deployment`

also small nit: Maybe add "Download will start when you close this dialogue" or something to the message as well? It wasn't clear to me as a user what to expect the first time when the dialogue popped up after I hit OK.

Reply:
I ended up needing to add some info to the dialog and change it to a `confirm()` so that folks can decide **not** to download it.